visual-java/src/main/java/com/saucelabs/visual/model/WindowScroll.java

```diff
@@ -0,0 +1,11 @@
+package com.saucelabs.visual.model;
+
+public class WindowScroll {
+  public int x;
```

Comment:
Could you please make these private and create getters? I think we only want to read them, right?

Reply:
I changed them to be `final`. Wdyt?

Reply:
I agree. Plus, we're already following the getter/setter pattern so it makes more sense to keep things aligned here.

Reply:
Changed to getters.

Reply:
Was this pushed? 🤔 I still see `.x` usage.